aci/resource_aci_cloudlb.go

```diff
@@ -277,6 +278,12 @@ func resourceAciCloudL4L7LoadBalancer() *schema.Resource {
 				Optional: true,
 				Set:      schema.HashString,
 			},
+			"static_ip_address": &schema.Schema{
```

Comment:
Should this attribute be added to the documentation of aci_cloud_l4_l7_native_load_balancer? When done in legacy docs, make sure to run go generate to have it correctly copied.

Reply:
added